x-pack/plugins/observability/public/pages/slos/components/slo_list_search_filter_sort_bar.tsx

```diff
@@ -27,10 +27,10 @@ export interface SloListSearchFilterSortBarProps {
   onChangeIndicatorTypeFilter: (filter: FilterType[]) => void;
 }
 
-export type SortType = 'name' | 'indicator_type';
+export type SortType = 'name' | 'indicatorType';
```

Comment:
💬 Missed part of a previous refactoring from snake_case to camelCase. The values have been updated to camelCase as well.